manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml

```diff
@@ -3,6 +3,9 @@ kind: ClusterRole
 metadata:
   name: scheduler-plugins-scheduler
 rules:
+- apiGroups: [""]
+  resources: ["namespaces", "configmaps"]
```

Comment:
remove configmaps as it's obtained via delegated authentication #272.

Reply:
I can still see errors in the logs that can't list namespaces